brainkeeper/src/app/app.component.ts

```diff
@@ -3,7 +3,7 @@ import { Component } from '@angular/core';
 @Component({
   selector: 'app-root',
   templateUrl: './app.component.html',
-  styleUrls: ['./app.component.scss']
+  styleUrls: ['./app.component.scss',],
```

Comment:
Just a note: Adding the trailing commas only makes sense when putting each entry in a new line. This reduces the amount of changed lines when adding a new entry. But when there is only one line you can leave them out. You don't have to change that, just keep it in mind